.circleci/build_dependencies.sh

```diff
@@ -47,6 +47,7 @@ comm -13 gendep_before.txt gendep_after.txt | grep -v gpg-pubkey | grep -v openQ
 # let's tidy if Tidy version changes
 newtidyver="$(git diff $thisdir/dependencies.txt | grep perl-Perl-Tidy | grep '^+' | grep -o '[0-9]*' || :)"
 [ -z "$newtidyver" ] || {
-    sed -i -e "s/\('Perl::Tidy',\s\+'==\s\)\([0-9]\+\)\(.*\)/\1$newtidyver\3/g" cpanfile
+    sed -i -e "s/\(Perl::Tidy):\s\+'==\s\)\([0-9]\+\)\(.*\)/\1$newtidyver\3/g" dependencies.yaml
+    docker exec -t gendep make update-deps
```

Comment:
Why is this needed now? :thinking:

Reply:
Because `make update-deps` will update the spec and cpanfile.

Reply:
I was missing the forest for the tree I guess :-P Yeah, that makes sense of course. Previously it was just patching the cpanfile directly